webapp/src/main/webapp/templates/freemarker/body/partials/individual/propStatement-dataDefault.ftl

```diff
@@ -15,7 +15,13 @@
 <@showStatement statement property datatype />
 
 <#macro showStatement statement property datatype>
+
     <#assign theValue = statement.value />
+
+<#-- create a weblink if the property range is a URI AND it contains http -->
+    <#if property.rangeDatatypeURI?? && theValue?contains("http")>
```

Comment:
```suggestion
    <#if property.rangeDatatypeURI?? && (datatype == "anyURI") && theValue?contains("http")>
```
Line 22 is only checking if the property's range URI exists, not confirming that the range is xsd:anyURI. With the current logic, any default data property with a range, e.g. xsd:string that happens to also contain 'http' in its value will be turned into a link. Attached is an example of this problem with the property 'hasRestriction.'
![screen shot 2018-11-06 at 2 58 52 pm](https://user-images.githubusercontent.com/10748475/48096802-714dae00-e1d5-11e8-950b-930cdc3b4db9.png)

I think my suggested logic will work as intended based on my interpretation. Note that it seems like only vcard properties have the anyURI datatype out of the box. They already have special templates so unless a user changes the range for an existing property or adds a new property with anyURI as a range, there won't be any difference in function.